### PR TITLE
Film profile

### PR DESCRIPTION
**Pull request by emilyb7**
Mostly templating stuff
- adds header to all views
- creates basic structure for film profile with menu (not working yet), image, themes, genre, summary
- lots of mixins to make styling easier across different sections of the site
- simple js for 'read more' links
related #25

### CODE REVIEW

File: templates/js/main.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+(function() {
+
+  function toggleDisplay(el) {
+    if (el.className.split(' ').indexOf('dn') > -1) {
+      return el.className = el.className.replace(' dn', '');
+    }
+
+    el.className = el.className + ' dn';
```
Comment (des-des) [inline templates/js/main.js]: Plus maybe a final `.trim()`

Comment (des-des) [reply templates/js/main.js]: Not really important

**Review by des-des — CHANGES_REQUESTED** [review]
okay cool, only thing is that tachyons has been committed. That may have been my bad.

I think you may have pull latest tachyons, and that wiped the code I put in tachyons/.gitignore

You could ignore it in the root .gitignore

File: templates/js/main.js (hunk 1 above)
Comment (des-des) [reply templates/js/main.js]: Also, you can use the module `class-names`

File: templates/js/main.js [hunk 2]
```diff
@@ -5,7 +5,7 @@
       return el.className = el.className.replace(' dn', '');
     }
 
-    el.className = el.className + ' dn'.trim();
+    (el.className = el.className + ' dn').trim();
```
Comment (des-des) [inline templates/js/main.js]: Brackets in wrong place
(el.className = el.className + ' dn').trim();
should be
el.className = (el.className + ' dn').trim();

**Review by des-des — CHANGES_REQUESTED** [review]

File: templates/js/main.js (hunk 2 above)
Comment (emilyb7) [reply templates/js/main.js]: ooops!

Comment (emilyb7) [reply templates/js/main.js]: fixed it

**Comment by emilyb7** [comment]
@des-des just made some extra changes, turns out it was all  a bit broken anyway...

File: templates/js/main.js [hunk 3]
```diff
@@ -15,10 +20,22 @@
     el.textContent = 'Show more';
   }
 
+  function filterByClassName(arr, className) {
+    return arr.find(function(node) {
```
Comment (des-des) [inline templates/js/main.js]: Array.find is es6, not supported by ie10

**Review by des-des — CHANGES_REQUESTED** [review]

File: templates/js/main.js (hunk 3 above)
Comment (emilyb7) [reply templates/js/main.js]: :'(

**Review by des-des — APPROVED** [review]

**Pull request merged**